api/envoy/config/route/v3/route_components.proto

```diff
@@ -1087,6 +1087,27 @@ message RetryPolicy {
     google.protobuf.Duration max_interval = 2 [(validate.rules).duration = {gt {}}];
   }
 
+  message RateLimitedRetryBackOff {
+    // Specifies the rate limited retry reset headers (like ``Retry-After`` or
+    // ``X-RateLimit-Reset``) to match against the response. If a request is
+    // being retried and the response contains one of these headers the value
+    // of the header will be used as the retry back off interval, with jitter
+    // added. If multiple headers are specified they are tried in the order
+    // given and the first matching header in the response is used.
+    //
+    // The value of the header must be either:
+    //  1. An interval in seconds as an integer, for example: ``120``.
+    //  2. A unix timestamp as an integer, for example: ``1595320702``. The timestamp
+    //     must be in the future relative to the current system time.
+    repeated HeaderMatcher reset_headers = 1 [(validate.rules).repeated = {min_items: 1}];
+
+    // Specifies the maximum rate limited retry back off interval that Envoy
+    // will allow. If a rate limited reset header contains an interval longer
+    // than this then the default exponential back off strategy will be used
+    // instead. Defaults to 300 seconds.
```

Comment:
Can you also clarify that this request is still governed by any global timeouts in place?

Reply:
Added to the docs